app/temporal/src/workflows/importer.workflow.ts

```diff
@@ -251,8 +261,26 @@ export async function importer(params: ImporterWorkflowParams) {
       },
     }
   );
+  setHandler(columnValidationUpdate, (params) => {
+    console.log("columnValidationUpdate", params);
+    columnConfig = columnConfig.map((column) => {
+      if (column.key === params.columnConfigKey) {
+        return {
+          ...column,
+          validations: column.validations?.map((validation) => {
```

Comment:
Wäre vielleicht prinzipiell eine Option sinnvoll wie canAddNewValues oder so? Vielleicht soll das nicht immer gehen bzw. ganz sicher soll das nicht immer gehen :)

Reply:
Ja müssten wir mal überlegen, wie wir das Konfigurieren - im FrontEnd hab ich schon ein Flag an der Component vorgesehen - is im Moment einfach immer True